### PR TITLE
docs(event-hubs): Provide authentication parameter for connection strings

### LINKED ISSUE #993: Connection string authentication is not documented for Azure Event Hubs
Connection string authentication is not documented for Azure Event Hubs when using trigger authentication.

https://keda.sh/docs/2.8/scalers/azure-event-hub/

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide authentication parameter for connection strings with Azure Event Hubs.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #993

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Thanks for the input @v-shenoy - Let me know if the version parity is correct.

**Comment by v-shenoy** [comment]
> Thanks for the input @v-shenoy - Let me know if the version parity is correct.

As far as I can see, even all the way back in v2.0.0 both `connection` and `storageConnection` are checked in auth params.

**Comment by tomkerkhove** [comment]
Done!

File: content/docs/2.0/scalers/azure-event-hub.md [hunk 1]
```diff
@@ -34,7 +34,16 @@ triggers:
 
 ### Authentication Parameters
 
-Not supported yet.
+You can authenticate by using connection string authentication.
+
+**Connection String Authentication:**
+
+- `connection` - Connection string for the Azure Service Bus Namespace.
```
Comment (v-shenoy) [inline content/docs/2.0/scalers/azure-event-hub.md:41]:
```suggestion
- `connection` - Connection string for the Azure Event Hub.
```

File: content/docs/2.0/scalers/azure-event-hub.md [hunk 2]
```diff
@@ -34,7 +34,16 @@ triggers:
...
+  
+  The following formats are supported.
+  
+  - With **SharedAccessKey** - `Endpoint=sb://<sb>.servicebus.windows.net/;SharedAccessKeyName=<key name>;SharedAccessKey=<key value>`
```
Comment (v-shenoy) [inline content/docs/2.0/scalers/azure-event-hub.md:45]:
```suggestion
  - With **SharedAccessKey** - `Endpoint=sb://<sb>.servicebus.windows.net/;SharedAccessKeyName=<key name>;SharedAccessKey=<key value>;EntityPath=<hub-name>`
```

File: content/docs/2.9/scalers/azure-event-hub.md [hunk 3]
```diff
@@ -61,13 +61,26 @@ triggers:
...
 ### Authentication Parameters
 
-The common way of authenticating to Azure Event Hub is by using the connection string. However, you can use [Azure AD Pod Identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) or [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/) providers if you host your cluster in Azure AKS, and if have configured it to support Pod Identity.
+You can authenticate by using pod identity or connection string authentication.
 
-To use Pod Identity, you have to add a [TriggerAuthentication](../concepts/authentication.md#re-use-credentials-and-delegate-auth-with-triggerauthentication) and configure it to use Pod Identity like so:
+**Connection String Authentication:**
 
-```
+- `connection` - Connection string for the Azure Service Bus Namespace.
```
Comment (v-shenoy) [inline content/docs/2.9/scalers/azure-event-hub.md:71]:
```suggestion
- `connection` - Connection string for the Azure Event Hub / Azure Event Hub Namespace.
```

File: content/docs/2.9/scalers/azure-event-hub.md [hunk 4]
```diff
@@ -61,13 +61,26 @@ triggers:
...
+  
+  The following formats are supported.
+  
+  - With **SharedAccessKey** - 
+    `Endpoint=sb://<sb>.servicebus.windows.net/;SharedAccessKeyName=<key name>;SharedAccessKey=<key value>`
```
Comment (v-shenoy) [inline content/docs/2.9/scalers/azure-event-hub.md:76]: Not sure what the best way to document this is. But the connection string can optionally end with EntityPath=<hub-name>. If it doesn't then the eventHubName parameter has to be provided (this has been added in the parameters section).

**Review by v-shenoy — CHANGES_REQUESTED** [review]

File: content/docs/2.0/scalers/azure-event-hub.md (hunk 1 above)
Comment (v-shenoy) [reply content/docs/2.0/scalers/azure-event-hub.md:41]: This applies for all versions 2.0.x => 2.8.x.

File: content/docs/2.0/scalers/azure-event-hub.md (hunk 2 above)
Comment (v-shenoy) [reply content/docs/2.0/scalers/azure-event-hub.md:45]: This applies for all versions 2.0.x => 2.8.x.

File: content/docs/2.0/scalers/azure-event-hub.md (hunk 1 above)
Comment (tomkerkhove) [reply content/docs/2.0/scalers/azure-event-hub.md:41]: Even copy pasting is something I'm not good at

File: content/docs/2.9/scalers/azure-event-hub.md (hunk 4 above)
Comment (tomkerkhove) [reply content/docs/2.9/scalers/azure-event-hub.md]: Done, let me know what you think!

File: content/docs/2.0/scalers/azure-event-hub.md (hunk 2 above)
Comment (tomkerkhove) [reply content/docs/2.0/scalers/azure-event-hub.md]: Done

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**